subsys/bluetooth/controller/Kconfig

```diff
@@ -68,13 +71,23 @@ choice BT_LL_CHOICE
 	help
 	  Select the Bluetooth Link Layer to compile.
 
-config BT_LL_SW_SPLIT
+config BT_LL_SW_SPLIT_LEGACY
 	bool "Software-based BLE Link Layer"
+	select BT_LL_SW_SPLIT
 	select BT_RECV_IS_RX_THREAD
 	select ENTROPY_GENERATOR
 	help
 	  Use Zephyr software BLE Link Layer ULL LLL split implementation.
 
+config BT_LL_SW_SPLIT_LLCP_SM
+	bool "Software-based BLE Link Layer with state-machine for control procedures"
+	select BT_LL_SW_SPLIT
+	select BT_RECV_IS_RX_THREAD
+	select ENTROPY_GENERATOR
+	help
+	  Use Zephyr software BLE Link Layer ULL LLL split implementation
+	  with a state-machine for the control procedures
+
```

Comment:
I'm pretty sure we already discussed this multiple times, we should only have one Kconfig option, not two that needs to be of opposite values.

Reply:
Agreed